### PR TITLE
Nest review comments under reviews.

### PR DESCRIPTION
**Pull request by grokys**
Previously we were loading a flat list of PR review comments into the `PullRequestModel.ReviewComments` property because this is how they were exposed in the REST API. 

GraphQL nests review comments under their relevant review, which makes a lot more sense, so this PR removes `PullRequestModel.ReviewComments` and moves them to `PullRequestReviewModel.Comments`.

This is the first step towards GraphQL-ification of Pull Requests; in upcoming PRs I intend to further this process by reading pull request models using purely GraphQL. At this point there are still a few hacks in place, and paging of PR reviews and review comments is not implemented. This will be addressed in upcoming PRs.

### CODE REVIEW

File: src/GitHub.InlineReviews/Services/IPullRequestSessionService.cs [hunk 1]
```diff
@@ -289,8 +289,8 @@ public interface IPullRequestSessionService
         /// <param name="number">The pull request number.</param>
         /// <param name="body">The comment body.</param>
         /// <param name="inReplyTo">The comment ID to reply to.</param>
-        /// <returns>A model representing the posted comment.</returns>
-        Task<IPullRequestReviewCommentModel> PostStandaloneReviewCommentRepy(
+        /// <returns>A model representing the review for the posted comment.</returns>
+        Task<IPullRequestReviewModel> PostStandaloneReviewCommentRepy(
```
Comment (StanleyGoldman) [inline src/GitHub.InlineReviews/Services/IPullRequestSessionService.cs]: `Repy` should probably be `Reply`

**Comment by grokys** [comment]
@StanleyGoldman could I get a 👍 on this? Fixed the misspelling.

File: src/GitHub.App/Models/PullRequestModel.cs [hunk 2]
```diff
@@ -177,18 +177,6 @@ public IReadOnlyList<IPullRequestReviewModel> Reviews
...
-            get { return reviewComments; }
-            set
-            {
-                Guard.ArgumentNotNull(value, nameof(value));
-                reviewComments = value;
-                this.RaisePropertyChange();
-            }
-        }
-
         IAccount assignee;
```
Comment (sguthals) [inline src/GitHub.App/Models/PullRequestModel.cs]: Why did you remove that?

**Comment by meaghanlewis** [comment]
Tested and LGTM 👍

**Comment by grokys** [comment]
Superseded by #1712.
